### DISCUSSION TITLE
notify-push Conainer hangs on starting

### DISCUSSION THREAD

**Discussion by davidcrds** [nextcloud/all-in-one#3710] [Questions] [answered]
Hi there, I'm trying to migrate from the nextcloud/docker Image to the AIO Image, everything starts up and nextcloud is available, but the notify-push container is in a constant restarting loop that ends with the message `/start.sh: line 51: /nextcloud/custom_apps/notify_push/bin/x86_64/notify_push: Permission denied`.

Initally I had my config in Portainer Stacks but since then I also tried running docker compose natively with and without root and running it with `docker run` as superuser. The result is always the same.

The only thing I found online was [this](https://help.nextcloud.com/t/aio-notify-push-container-does-not-start-permission-denied/171600), but it did not work for me.

Any tips are appreciated!

**Comment by szaimen** [comment, marked as answer]
This will be fixed with https://github.com/nextcloud/all-in-one/pull/3707

**Reply by szaimen** [reply]
This is now fixed with 7.7.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by davidcrds** [reply]
Yes, the newest beta released fixed the starting issue for me. All containers are running now. Thanks!
